external/CMakeLists.txt

```diff
@@ -185,3 +185,43 @@ target_include_directories(breakpad_client
     PUBLIC
     "$<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}>"
 )
+
+target_compile_features(breakpad_client PUBLIC cxx_std_11)
+if(CMAKE_SYSTEM_NAME STREQUAL "Linux" OR CMAKE_SYSTEM_NAME STREQUAL "FreeBSD")
+    find_path(LINUX_SYSCALL_INCLUDE_DIR NAMES linux_syscall_support.h)
+    target_include_directories(breakpad_client PRIVATE ${LINUX_SYSCALL_INCLUDE_DIR})
+endif()
```

Comment:
Please read my bigger comment for why a simple replacement like this and your `breakpad` PR will not be accepted.